### DISCUSSION TITLE
moving Repository: Archive: keep columns "size" and "time"

### DISCUSSION THREAD

**Discussion by Golddouble** [borgbase/vorta#843] [Ideas]
**Is your feature request related to a problem? Please describe.**
When I move my Repository to another place, I use "Disconnect repository". Then I move my Folder that contains the Backup to another place. Then I go to "Add existing repository" and choose the new place.
Then I go to the "archive" tab. -> In the size and time column all infos are deleted.

**Describe the solution you'd like**
Save this information also in the folder of the backup, so that this infos can be brought back again.

**Comment by samuel-w** [comment]
The issue is that it could cause deleted repositories to take up space in the database.

Some possible solutions:

- Dialog box to see if user wants to delete data
       Kind of bad, since there is no way for user to delete data later. I also prefer keeping dialog boxes to a minimum.
- Keep data, and delete it after N days, similar to old logs
       Have to store deletion date in DB, cannot use existing archive time attribute.

**Comment by Golddouble** [comment]
Where exactly are this data (size and time) saved at the moment? In which path?

**Comment by samuel-w** [comment]
They are stored in the database, specifically archivetablemodel. The database is in `~/.local/share/Vorta` on Linux.

**Comment by Golddouble** [comment]
Thank you. But I have not found this folder in my flatpack version on Linux. Is it in another folder in the flatpack version?

**Comment by samuel-w** [comment]
For flatpak, it is `~/.var/app/com.borgbase.Vorta/data/Vorta`
